Inner and suppressed exception support on Android (#3028)
saucelabs/sauce-docs: docs/error-reporting/platform-integrations/android/configuration.md

</TabItem>
</Tabs>

### Sending Inner and Suppressed exceptions

:::warning Inner and suppressed exception support

> The new inner and suppressed exception support is available starting backtrace-android@3.9.0

Reviewer: ```suggestion
Support for inner and suppressed exceptions is available starting backtrace-android@3.9.0
```

:::

> Backtrace clients allow sending Inner and suppressed exceptions. By default, this option is disabled. To enable inner and suppressed exception support, use the code shown below.

Reviewer: ```suggestion
Backtrace client allows sending Inner and suppressed exceptions. By default, this option is disabled. To enable inner and suppressed exception support, use the code shown below.
```
